Sources/Common/Store/GlobalDataStore.swift

```diff
@@ -16,23 +11,7 @@ public protocol GlobalDataStore: AutoMockable {
 
 // sourcery: InjectRegister = "GlobalDataStore"
 public class CioGlobalDataStore: GlobalDataStore {
-    private var diGraph: DIGraph {
-        // Used *only* for information that needs to be global between all site ids!
-        DIGraph.getAllWorkspacesSharedInstance()
-    }
-
-    internal var keyValueStorage: KeyValueStorage {
-        diGraph.keyValueStorage
-    }
-
-    public var sharedInstanceSiteId: String? {
-        get {
-            keyValueStorage.string(.sharedInstanceSiteId)
-        }
-        set {
-            keyValueStorage.setString(newValue, forKey: .sharedInstanceSiteId)
-        }
-    }
+    private let keyValueStorage: KeyValueStorage
```

Comment:
`private` is private to the class. `internal` is scoped to the module. 

I was using `internal` so automated test code can access it to delete data between tests. But,this felt like a hack to me. I improved how tests delete data so this can now be `private`.